qutip/models/tests/test_dicke.py

```diff
@@ -0,0 +1,309 @@
+"""
```

Comment:
why not put this file in qutip/tests. Will this file still be picked up by nose?

Reply:
Is there an answer for this?